.circleci/config.yml

```diff
@@ -33,6 +33,17 @@ jobs:
       - run:
           name: Test login page HTTP status
           command: curl -f http://localhost:8080/
+      - run:
+            name: Shutdown prod mode
+            command: make down
+      - run:
+            name: Install PHP dependencies - PIM dev
+            command: docker run -u www-data -u www-data -v $(pwd):/srv/pim -w /srv/pim --rm akeneo/pim-php-dev:4.0 php -d memory_limit=4G /usr/local/bin/composer require akeneo/pim-community-dev 4.0.x-dev
+      - run:
+            name: Launch the PIM in dev mode
+            command: make
```

Comment:
I wouldn't recommend using just make
(in case the default command change)
also, it's not explicit at all imo

Reply:
Agree, but i used it to be consistent with existing steps.